library/core/src/macros/mod.rs

```diff
@@ -868,6 +868,11 @@ pub(crate) mod builtin {
     /// let s = fmt::format(format_args!("hello {}", "world"));
     /// assert_eq!(s, format!("hello {}", "world"));
     /// ```
+    ///
+    /// # Known issue
+    ///
+    /// There is a limitation about the lifetime returned by `format_args!`,
+    /// see [#92698](https://github.com/rust-lang/rust/issues/92698).
```

Comment:
"the lifetime returned by" doesn't sound right. Lifetimes are not "returned".

I think there should be a bit more context here to explain the limitation, so readers don't have to go check out that github issue to know what the limitation is.

Maybe something like:

```suggestion
    /// # Lifetime limitation
    ///
    /// Except when no formatting arguments are used,
    /// the produced `fmt::Arguments` value borrows temporary values,
    /// which means it can only be used within the same expression
    /// and cannot be stored for later use.
    /// This is a known limitation, see [#92698](https://github.com/rust-lang/rust/issues/92698).
```

Reply:
haha my english is not great, thx for your time, change & rebase done.